test-framework/test-suites/system/files/wait-for-backend.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+for i in {1..60}
+do
+    if grep $1 /var/log/checklist.log | grep -q 'TFTP_RRQ'
```

Comment:
we're unlikely to have 10 vm's in this, but it would potentially break here, right?

Reply:
Not sure I understand what you are seeing. The $1 will the an IP address.

Reply:
I think I get you now. You are wondering if sleeping for 10 minutes is enough. It is currently, because we currently only have 9 VMs max (3 integration, 4 system, 1 unit, 1 combine coverage). However, there is no hurt in increasing this number, so I will.

Reply:
Not what I meant, but maybe I'm misunderstanding.  This code will grep for an ip, say `192.168.0.1`.  But that will also match `192.169.0.10`.  We don't have that ip assigned right now, but if we did?  Probably just over-thinking it.

Reply:
Oh, IC. Yah, if system tests fired up a 9th backend (since the frontend takes an IP) then my grep wouldn't work. I could rewrite the grep in that case to be more specific, but for now I don't see us ever having that many backends.

Reply:
Yup fair enough.  ✅  :shipit: